packages/beacon-node/src/api/impl/node/index.ts

```diff
@@ -63,11 +63,11 @@ export function getNodeApi(
     },
 
     async getSyncingStatus() {
-      return {data: sync.getSyncStatus()};
+      return {data: await sync.getSyncStatus()};
     },
 
     async getHealth(_req, res) {
-      if (sync.getSyncStatus().isSyncing) {
+      if ((await sync.getSyncStatus()).isSyncing) {
```

Comment:
How often are these called? and will this effect validator performance that now each response will likely take twice as long as before (because each call now awaits a response by the EL)?